plugins/module_utils/network/sonic/config/sFlow/sFlow.py

```diff
@@ -0,0 +1,376 @@
+        else:
+            commands = []
+        return commands, requests
+
+    def contains_interface(self, list, search_interface):
+        for interface in list:
+            if interface["name"] == search_interface["name"]:
+                return True
+        return False
+
+    def contains_collector(self, list, search_collector):
+        for collector in list:
+            if collector["address"] == search_collector["address"] and collector["network_instance"] == search_collector["network_instance"]\
+                and collector["port"] == search_collector["port"]:
+                return True
+        return False
+
+    def validate_sflow_args(self, config):
+        '''validates passed in config'''
+        validated = utils.validate_config(self._module.argument_spec, {"config":config})
+        if config is not None and "polling_interval" in config and config["polling_interval"] is not None:
+            if not (int(config["polling_interval"]) == 0 or int(config["polling_interval"]) in range(5,300)):
+                raise Exception("polling interval out of range. must be 0 or in [5:300]")
+    
+    def create_patch_sFlow_root_request(self, config_dict, request_list):
+        '''builds REST request for patching on sflow root, which can update all sflow information in one REST request, 
+        from given config. adds request to passed in request list and returns it'''
+        method = "PATCH"
+        root_data_key = "openconfig-sampling-sflow:sflow"
+
+        if len(config_dict) == 0:
+            return request_list
+
+        request_body = {}
+
+        has_data = False
+
+        #config always requred in this endpoint
+        request_body["config"] = self.create_config_request_body(config_dict)
+        if len(request_body["config"]) > 0:
+            has_data = True
+
+        if "collectors" in config_dict:
+            collector_body = self.create_collectors_list_request_body(config_dict)
+            if len(collector_body) > 0:
+                request_body.update({"collectors":{"collector":collector_body}})
+                has_data = True
+
+        if "interfaces" in config_dict:
+            interface_body = self.create_interface_list_request_body(config_dict)
+            if len(interface_body) > 0:
+                request_body.update({"interfaces":{"interface":interface_body}})
+                has_data = True
+
+        if has_data:
+            request_list.append({"path": self.sflow_uri, "method": method, "data": {root_data_key:request_body}})
+        return request_list
+    
+    def create_config_request_body(self, config_dict):
+        '''creates the REST API format sflow global config info. '''
+        request_config = {}
+        if "enabled" in config_dict:
+            # request_list.append({"path":"data/openconfig-sampling-sflow:sampling/sflow/config/enabled", "method":"PUT", "data":{"openconfig-sampling-sflow:enabled":config_dict["enabled"]}})
```

Comment:
Delete code rather than commenting it out.

Reply:
oof forgot about that after testing out changes.